src/pages/visualizations/[visualizationId].tsx

```diff
@@ -0,0 +1,159 @@
+import Block from "core/components/Block";
+import Breadcrumbs from "core/components/Breadcrumbs";
+import Button from "core/components/Button";
+import DataCard from "core/components/DataCard";
+import CountryProperty from "core/components/DataCard/CountryProperty";
+import DateProperty from "core/components/DataCard/DateProperty";
+import RenderProperty from "core/components/DataCard/RenderProperty";
+import { OnSaveFn } from "core/components/DataCard/Section";
+import TagProperty from "core/components/DataCard/TagProperty";
+import TextProperty from "core/components/DataCard/TextProperty";
+import UserProperty from "core/components/DataCard/UserProperty";
+import Page from "core/components/Layout/Page";
+import { PageContent } from "core/components/Layout/PageContent";
+import Spinner from "core/components/Spinner";
+import { ensureArray } from "core/helpers/array";
+import { createGetServerSideProps } from "core/helpers/page";
+import useCacheKey from "core/hooks/useCacheKey";
+import useToggle from "core/hooks/useToggle";
+import { useTranslation } from "next-i18next";
+import { useState } from "react";
+import VisualizationPicture from "visualizations/features/VisualizationPicture";
+import { VisualizationPicture_VisualizationFragment } from "visualizations/features/VisualizationPicture.generated";
+import {
+  useVisualizationQuery,
+  VisualizationDocument,
+  VisualizationQuery,
+  VisualizationsPageQuery,
+} from "visualizations/graphql/queries.generated";
+
+type Props = {
+  visualizationId: string;
+};
+
+const CollectionPage = ({ visualizationId }: Props) => {
+  const { t } = useTranslation();
+
+  const { data, refetch } = useVisualizationQuery({
+    variables: { id: visualizationId },
+  });
+
+  useCacheKey("collections", () => refetch());
+
+  if (!data?.externalDashboard) {
+    return null;
+  }
+
+  const { externalDashboard } = data;
+
+  return (
+    <Page title={externalDashboard.name}>
+      <PageContent>
+        <Breadcrumbs className="my-8 px-2">
+          <Breadcrumbs.Part href="/visualizations">
+            {t("External Dashboard")}
+          </Breadcrumbs.Part>
+          <Breadcrumbs.Part
+            href={{
+              pathname: "/visualizations/[visualizationId]",
+              query: { visualizationId: externalDashboard.id },
+            }}
+          >
+            {externalDashboard.name}
+          </Breadcrumbs.Part>
+        </Breadcrumbs>
+        <div className="space-y-10">
+          <DataCard item={externalDashboard}>
+            <DataCard.Heading<typeof externalDashboard> titleAccessor="name">
+              {() => (
```

Comment:
It's better to used `item` instead of using the `externalDashboard` object as the `item` is the one used in the data card

```suggestion
              {(item) => (
```